packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -37,4 +45,58 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
     function getMemberTokenAddress() public view override returns (address) {
         return _electionStore().memberTokenAddress;
     }
+
+    function getNominees() external view override returns (address[] memory) {
+        return _electionStore().nominees;
+    }
+
+    function nominate() external override {
+        ElectionStore storage store = _electionStore();
+
+        if (msg.sender == address(0)) {
+            revert AddressError.ZeroAddress();
+        }
+
+        if (store.nomineeIndexes[msg.sender] != 0) {
+            revert AlreadyNominated(msg.sender);
+        }
+
+        store.nominees.push(msg.sender);
+        store.nomineeIndexes[msg.sender] = store.nominees.length;
+    }
+
+    function withdrawNomination() external override {
+        ElectionStore storage store = _electionStore();
+
+        uint256 valueIndex = store.nomineeIndexes[msg.sender];
+
+        if (msg.sender == address(0)) {
```

Comment:
Move this above the index read?

Reply:
this condition can be removed (unreachable path)

Reply:
Done!